src/main/java/uk/gov/hmcts/befta/util/BeftaUtils.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.befta.util;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class BeftaUtils {
+
+    public static File getClassPathResourceIntoTemporaryFile(String resourcePath) {
+        try {
+            InputStream stream = BeftaUtils.class.getClassLoader().getResource(resourcePath).openStream();
+            byte[] buffer = new byte[stream.available()];
```

Comment:
that will cause a Runtime Error, but only for really large files that challenge the allocated memory of the VM running the tests. So, for now, I would think it will not be a concern.